src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyReference.cs

```diff
@@ -10,12 +10,12 @@ public KeyReference() : base()
             ReferenceType = "KeyReference";
         }
 
-        public KeyReference(string uri) : base(uri)
+        public KeyReference([System.Diagnostics.CodeAnalysis.StringSyntaxAttribute("Uri")] string uri) : base(uri)
         {
             ReferenceType = "KeyReference";
         }
 
-        public KeyReference(string uri, TransformChain transformChain) : base(uri, transformChain)
+        public KeyReference([System.Diagnostics.CodeAnalysis.StringSyntaxAttribute("Uri")] string uri, TransformChain transformChain) : base(uri, transformChain)
```

Comment:
I suspect CI is going to fail when trying to build the netstandard2.0 / .NET Framework versions of this binary, as StringSyntaxAttribute doesn't exist there. The project file would need to be updated to pull the source for the attribute in as is done in other similar projects, e.g.
https://github.com/dotnet/runtime/blob/0d863807580bae82225e5b7159c488a877530a74/src/libraries/Microsoft.Extensions.Logging.Console/ref/Microsoft.Extensions.Logging.Console.csproj#L17

Reply:
We shouldn't be including this file at all for .NETFramework - we type forward there and do not expose new API.  For .NETStandard it can - in both src and ref.